src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -239,7 +239,7 @@ public class Ado : NotificationsBase, IAdo {
 
         var renderedConfig = RenderAdoTemplate(logTracer, renderer, config, instanceUrl);
         var ado = new AdoConnector(renderedConfig, project!, client, instanceUrl, logTracer, await GetValidFields(client, project));
-        await ado.Process(notificationInfo);
+        await ado.Process(notificationInfo, config.AdoDuplicateFields);
```

Comment:
The `AdoConnector` already has an instance of the rendered config so we actually don't even need to modify the `Process` function signature.